ui/client/components/graph/node-modal/editors/expression/BoolEditor.tsx

```diff
@@ -1,17 +1,23 @@
 import i18next from "i18next"
 import {isEmpty} from "lodash"
-import React, {ReactNode} from "react"
-import ExpressionWithFixedValues from "./FixedValuesEditor"
+import React from "react"
+import FixedValuesEditor from "./FixedValuesEditor";
+
+type BoolEditor<P> = React.FunctionComponent<P> & {
```

Comment:
isn't this generic type for SimpleEditor?

Reply:
No, this is a concrete implementation of simple editor, one of many. Not always there will be FixedValuesEditor inside.